be/src/storage/tablet_updates.cpp

```diff
@@ -951,7 +953,7 @@ void TabletUpdates::_apply_rowset_commit(const EditVersionInfo& version_info) {
         }
         // 4. write meta
         st = TabletMetaManager::apply_rowset_commit(_tablet.data_dir(), tablet_id, _next_log_id, version, new_del_vecs,
-                                                    index_meta, _tablet.get_enable_persistent_index());
+                                                    index_meta, enable_persistent_index);
```

Comment:
can use index.enable_persistent_index() directly instead of an yet another individual variable.